scanner.c is generated by flex using scanner.l, scanner.l needs gram.h
when produce scanner.c, so redefine the dependency relation

Upstream-Status: Pending

Signed-off-by: Roy.Li <rongqing.li@windriver.com>
---
 Makefile.am |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 04b834b..2da986c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -131,5 +131,4 @@ dist-hook:
 	rm -f $(distdir)/gram.h
 	rm -f $(distdir)/scanner.c
 
-scanner.o: gram.h
-
+scanner.c: gram.h
-- 
1.7.4.1

